### PR TITLE
fix: link title matching text

### PR DESCRIPTION
**Pull request by nschonni**
Find: ` title="(\w*)">(\1)</a>`
Replace: `>$2</a>`

### CODE REVIEW

File: files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h2 id="Audio_buffers_frames_samples_and_channels">Audio buffers: frames, sample
...
+ Secondly, signals must be <a href="https://en.wikipedia.org/wiki/Low-pass_filter" title="Low-pass filter">low-pass filtered</a> before sampling, otherwise <a href="https://en.wikipedia.org/wiki/Aliasing">Aliasing</a> occurs. While an ideal low-pass filter would perfectly pass frequencies below 20 kHz (without attenuating them) and perfectly cut off frequencies above 20 kHz, in practice a <a href="https://en.wikipedia.org/wiki/Transition_band" title="Transition band">transition band</a> is necessary, where frequencies are partly attenuated. The wider this transition band is, the easier and more economical it is to make an <a href="https://en.wikipedia.org/wiki/Anti-aliasing_filter" title="Anti-aliasing filter">anti-aliasing filter</a>. The 44.1 kHz sampling frequency allows for a 2.05 kHz transition band.</p>
```
Comment (vkWeb) [inline files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html:92]: Lowercase `A` in Aliasing.

File: files/en-us/web/api/window/index.html [hunk 2]
```diff
@@ -31,7 +31,7 @@ <h2 id="Constructors">Constructors</h2>
 
 <dl>
  <dt>{{domxref("DOMParser")}}</dt>
- <dd><code>DOMParser</code> can parse XML or HTML source stored in a string into a DOM <a href="/en-US/docs/DOM/document" title="document">Document</a>. <code>DOMParser</code> is specified in <a href="https://w3c.github.io/DOM-Parsing/" title="http://html5.org/specs/dom-parsing.html">DOM Parsing and Serialization</a>.</dd>
+ <dd><code>DOMParser</code> can parse XML or HTML source stored in a string into a DOM <a href="/en-US/docs/DOM/document">document</a>. <code>DOMParser</code> is specified in <a href="https://w3c.github.io/DOM-Parsing/" title="http://html5.org/specs/dom-parsing.html">DOM Parsing and Serialization</a>.</dd>
```
Comment (vkWeb) [inline files/en-us/web/api/window/index.html:34]: Uppercase d in document needed.

File: files/en-us/glossary/canvas/index.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
...
+<p><span class="seoSummary">The <strong>canvas element</strong> is part of <a href="https://en.wikipedia.org/wiki/HTML5">HTML5</a> and allows for dynamic, <a href="https://en.wikipedia.org/wiki/Scripting_language" title="Scripting language">scriptable</a> <a href="https://en.wikipedia.org/wiki/Rendering_(computer_graphics)" title="Rendering (computer graphics)">rendering</a> of 2D and 3D shapes and <a href="https://en.wikipedia.org/wiki/Bitmap">Bitmap</a> images.</span> It is a low level, procedural model that updates a <a href="https://en.wikipedia.org/wiki/Bitmap">Bitmap</a> and does not have a built-in <a href="https://en.wikipedia.org/wiki/Scene_graph" title="Scene graph">scene graph</a>. It provides an empty graphic zone on which specific {{Glossary("JavaScript")}} {{Glossary("API","APIs")}} can draw (such as Canvas 2D or {{Glossary("WebGL")}}).</p>
```
Comment (vkWeb) [inline files/en-us/glossary/canvas/index.html:11]: please lowercase b in Bitmap.

File: files/en-us/learn/css/howto/css_faq/index.html [hunk 4]
```diff
@@ -186,7 +186,7 @@ <h3 id="Use_of_the_.2A_selector">Use of the <code>*</code> selector</h3>
 
 <h3 id="Specificity_in_CSS">Specificity in CSS</h3>
 
-<p>When multiple rules apply to a certain element, the rule chosen depends on its style <a href="/en-US/docs/Learn/CSS/Introduction_to_CSS/Cascade_and_inheritance#Specificity" title="Specificity">specificity</a>. Inline style (in HTML <code>style</code> attributes) has the highest specificity and will override any selectors, followed by ID selectors, then class selectors, and eventually element selectors. The text color of the below {{htmlelement("div")}} will therefore be red.</p>
+<p>When multiple rules apply to a certain element, the rule chosen depends on its style <a href="/en-US/docs/Learn/CSS/Introduction_to_CSS/Cascade_and_inheritance#Specificity">Specificity</a>. Inline style (in HTML <code>style</code> attributes) has the highest specificity and will override any selectors, followed by ID selectors, then class selectors, and eventually element selectors. The text color of the below {{htmlelement("div")}} will therefore be red.</p>
```
Comment (vkWeb) [inline files/en-us/learn/css/howto/css_faq/index.html:189]: Lowercase S in Specificity.

**Review by vkWeb — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html:92]: Good call, i'll see if I can do a backwards match for the regex to keep the casing

**Comment by nschonni** [comment]
@vkWeb was able to change the regex to use the link text for the final replacement so it just drops the `title`

**Review by vkWeb — APPROVED** [review]
Now this looks good. As always nice work, thanks @nschonni. :smile:

**Review by chrisdavidmills — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
Thanks @nschonni , and thanks @vkWeb for the review!

**Pull request merged**